Makefile

```diff
@@ -202,7 +202,7 @@ eslint: node-babel-eslint node-eslint node-eslint-plugin-html
 
 dev_data:
 	cd $(WPTD_GO_PATH)/util; go get -t ./...
-	go run $(WPTD_GO_PATH)/util/populate_dev_data.go $(FLAGS)
+	go run $(WPTD_GO_PATH)/util/populate_dev_data.go $(FLAGS) "-static_dir=$(WPTD_GO_PATH)/webapp/static"
```

Comment:
Nit: technically I think this should be:

`... -static_dir="$(WPTD_GO_PATH)/webapp/static"`

(In reality it probably doesn't matter.)

Reply:
@mdittmer ahh sorry I was perhaps not clear enough. I meant the location of the quote was wrong. Usually we only quote the value of a flag but not the whole flag, because we know the flag name and it won't contain spaces. When someone quotes the whole flag (including the flag name), it suggests the intention of passing the whole flag as a string through a wrapper. Here it doesn't really make a difference but I was nitpicking.

I modified the PR in place for you.